TensorLib/Test.lean

```diff
@@ -27,10 +27,6 @@ private def testTensorElementBV (dtype : Dtype) : IO Bool := do
   let expected := (Tensor.arange! dtype 20).reshape! (Shape.mk [5, 4])
   return Tensor.arrayEqual expected arr
 
-#eval do
-  let b <- testTensorElementBV Dtype.uint16
```

Comment:
This test runs during `tensorlib test` below, so doesn't need to be evaluated also at build time.